configdist.py

```diff
@@ -1,17 +1,18 @@
 # Database config
 DATABASE_URL = "sqlite:///test.db"
 
-TWILIO_ACCOUNT_SID = ''
-TWILIO_ACCOUNT_AUTH_TOKEN = ''
-TWILIO_FROM_NUMBER = ''
+TWILIO_ACCOUNT_SID = ""
+TWILIO_ACCOUNT_AUTH_TOKEN = ""
+TWILIO_FROM_NUMBER = ""
+TWILIO_TEST_NUMBER = ""
```

Comment:
Should probably put in a comment here about how the number needs to be `+1XXXXXXXXXX`.